roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/util/Test.java

```diff
@@ -0,0 +1,84 @@
+package at.fhv.itm3.s2.roundabout.util;
+
+import at.fhv.itm14.trafsim.model.entities.IConsumer;
+import at.fhv.itm3.s2.roundabout.RoundaboutSimulationModel;
+import at.fhv.itm3.s2.roundabout.api.entity.ConsumerType;
+import at.fhv.itm3.s2.roundabout.entity.*;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class Test {
```

Comment:
why class test in main and not in test directory?

Reply:
maybe roundabout-core/src/test/java/at/fhv/itm3/s2/roundabout/util/ParserTest.java?